src/pages/resultsView/plots/PlotsTabUtils.tsx

```diff
@@ -597,23 +605,23 @@ export const mutationSummaryRenderPriority = stringListToIndexSet(mutationSummar
 const cnaToAppearance = {
     "-2":{
         legendLabel: "Deep Deletion",
-        stroke:"#00008B",
+        stroke:CNA_COLOR_HOMDEL,
     },
     "-1":{
         legendLabel: "Shallow Deletion",
-        stroke:"#00BFFF",
+        stroke:"#2aced4",
```

Comment:
these are slight variations of default coloring?

Reply:
they're slight variations (mainly, increased saturation) from the oncoprint colors

Reply:
@adamabeshouse might be nice to use sass instead, e.g. `saturate( $base-color, 20% )`